contracts/vesting.token/include/vesting.token/vesting.token.hpp

```diff
@@ -0,0 +1,90 @@
+// vesting.token.hpp
+
+#pragma once
+
+#include <eosio/action.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/asset.hpp>
+#include <eosio/system.hpp>
+#include <eosio/singleton.hpp>
+
+namespace vestingtoken {
+    using eosio::action_wrapper;
+    using eosio::asset;
+    using eosio::name;
+    using std::string;
+    using eosio::check;
+
+    eosio::time_point sales_start_date;
+    eosio::time_point launch_date;
+
+    static const uint32_t SECONDS_IN_DAY = 24*60*60;
+
+    struct vesting_category  {
+        uint32_t  cliff_period_days;
```

Comment:
lets change this to be seconds, that takes the computation burden out of the transactions, which is a bit more efficient